musicbot/playlist.py

```diff
@@ -344,11 +344,17 @@ def peek(self):
         """
             (very) Roughly estimates the time till the queue will 'position'
         """
-        estimated_time = sum(e.duration for e in islice(self.entries, position - 1))
+        if any(e.duration for e in islice(self.entries, position - 1) == None):
+            raise InvalidDataError('no duration data')
```

Comment:
We shouldn't return an error if no duration is found. Would be much more user friendly to return a string stating that no duration is found because its not an error, its an expected condition.

Reply:
I'd argue that raising an error here is better than returning a string and then the caller has to check for type returned to format accordingly. Hence, I modified the code in the play command to not raise an error but to change the reply message instead.

Reply:
Maybe make this a bit more wordy then? Less than a three word error so that way a user understands the bot isn’t broken from this error and they did nothing wrong, they just queued a song we couldn’t fetch the time info for

Reply:
Ah, i see you’re handing this in the create embed. I must have missed this the first time.